src/main/java/com/example/onlinebookstore/dto/book/BookDto.java

```diff
@@ -0,0 +1,17 @@
+package com.example.onlinebookstore.dto.book;
+
+import java.math.BigDecimal;
+import java.util.List;
+import lombok.Data;
+
+@Data
+public class BookDto {
```

Comment:
make it record

Reply:
I can't make it record, because I need to set its categoriesIds field value in CategoryMapper @AfterMapping